### PR TITLE
Textual updates

### LINKED ISSUE #71: bug report from ESA
Dear Bengt,

I hope everything is going fine with you. 

I am now back on Odin stuff.

At page http://odin.rss.chalmers.se/dataaccess 

I noticed that two of the provided Python code snippets indicate 85 km incorrectly -> "max_altitude": 850000.

Could you please have a look?

### LINKED ISSUE #72: Add planning document to website
On the final meeting ESA asked us to publish the planning document (excel-file) together with some comments on how to interpret it.

@donal-mur can you send us the planning document?

Also add PVER-document

### LINKED ISSUE #73: Publish warning for FM1 LO instability
ESA requested a warning on the webpage for a instrumental issue for years 2010-2017 with impact on FM1 data.

@donal-mur write a short note on what what message you want to publish as a comment in this issue.

### LINKED ISSUE #77: Version 3 release text
@donal-mur write a short "release note" for the wen site.

### PR DESCRIPTION
**Pull request by skymandr**
Resolves #77 
Resolves #72 
Resolves #71 
Resolves #73

### CODE REVIEW

File: src/odinapi/templates/index.html [hunk 1]
```diff
@@ -1,58 +1,135 @@
...
+      <div class="col-md-7">
+        <h2 class="featurette-heading">Warning for <em>FM&nbsp;01</em> <span class="text-muted">for 2010–2018</span></h2>
+        <p class="lead">
+          Users are warned that data from FM&nbsp;01 may be unreliable
+          between early 2010 and January 2018 due to a misbehaving local
+          oscillator in the instrument.
+        </p>
+        <p>
+          This has manifested itself by introducing asymmetric and broadened
+          line shapes resulting in too low ozone and ClO values. We are
```
[the remark this answers is not in the document]
Comment (skymandr) [reply src/odinapi/templates/index.html:82]: I thought so too, but I'm not responsible for the copy.

[the remark this answers is not in the document]
Comment (donal-mur) [reply src/odinapi/templates/index.html:82]: mm you can write carbon monoxide if you prefer

File: src/odinapi/templates/index.html [hunk 2]
```diff
@@ -1,58 +1,135 @@
...
-
-          <p>The satellite has been used as a true observatory in order to best
+          (aeronomy).
+        </p>
+        <p>
+          The satellite has been used as a true observatory in order to best
           respond to scientific challenges. The astronomy mission was
           successfully concluded in the spring of 2007 and since then the
           satellite has been used exclusively for studies of the Earth’s
           atmosphere. During the first three years of the Odin mission, when
```
Comment (donal-mur) [inline src/odinapi/templates/index.html:128]: should be 6 years 2001-2007

**Review by donal-mur — COMMENTED** [review]
Some small comments

**Review by local-minimum — APPROVED** [review]

**Pull request merged**